### ISSUE TITLE
what should the description be?

### ISSUE THREAD

**Issue by jaekwon** [atomone-hub/atom.one#1]
isn't atom.one about the website static content? 
just trying to make the purpose of this repo clear, 
and disambiguate from the other atomone repo.

**Comment by tbruyelle** [comment]
Yes, this repo contains the markdown files that will be used to serve the https://atom.one website content.

**Comment by tbruyelle** [comment]
@jaekwon I added this description and a link:
![image](https://github.com/user-attachments/assets/972d125c-54fe-41ae-9a58-76516bfd1f81)

**Comment by tbruyelle** [comment]
I think the issue has been addressed, if not please reopen it.
